Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -626,7 +626,7 @@ static void HandleRemoveBreakpoint()
   u32 type, addr, len, i;
 
   type = Hex2char(s_cmd_bfr[1]);
-  if (type >= 4)
+  if (type > 4)
```

Comment:
Could you turn this magic number into a named constant? You could add `constexpr u32 NUM_BREAKPOINT_TYPES = 4;` right after the BreakpointType enum (and then use that constant in HandleAddBreakpoint too).